src/js/componentBoilerplate.js

```diff
@@ -0,0 +1,3 @@
+function oComponentBoilerplate() {};
```

Comment:
probably worth adding a static init function https://github.com/Financial-Times/o-header/blob/master/src/js/header.js#L32 as it's executed in the `o.DOMContentLoaded` handler in main.js and every module should have a static `init()` function